### DISCUSSION TITLE
Hide "Data & Details" Header

### DISCUSSION THREAD

**Discussion by muksidin** [nocodb/nocodb#7825] [Features]
Hi, i think this is better to hide header "Data & Details" and this feature only for administrator.
![Screenshot_2](https://github.com/nocodb/nocodb/assets/5665115/30a1fe34-4ab5-4949-94db-b24f0e20db4c)

**Comment by dstala** [comment]
Details tab includes features that are essential for Creator - Multi field editor, Code snippets, Automations
Depending on the role, some of these will get disabled/hidden.

**Reply by PhilDu77** [reply]
tanks.
I can understand for a creator role but in my case it is more for un simple editor role use.
he needs only to see records or add a new one.
only Fields screen in the detail could be useful to set himself fields to show but Relations, APIs,Webhooks can loose simple editor user.

**Comment by PhilDu77** [comment]
hi,
I dont need to show the Details tab for simple editor users.
How can i desable this tab please ?
